ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -92,7 +92,6 @@ tests:
   steps:
     cluster_profile: alibabacloud-qe
     env:
-      E2E_RUN_TAGS: '@console'
```

Comment:
since we are removing the `E2E_RUN_TAGS` and we should add `E2E_TEST_TYPE` here?

Reply:
I see, the `E2E_TEST_TYPE` has default value `ui` for chain: openshift-e2e-test-qe-ui